.github/workflows/Specs.yml

```diff
@@ -0,0 +1,45 @@
+name: Specs
+
+jobs:
+  specs:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        ruby: [2.6.3]
+
+    name: ${{ matrix.os }} / Ruby ${{ matrix.ruby }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout git
+        uses: actions/checkout@v1
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        if: ${{ matrix.ruby != 'system' }}
+        with:
+          ruby-version: ${{ matrix.ruby }}
+
+      - name: Install packages
+        run: |
+          sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
```

Comment:
I got this from https://askubuntu.com/questions/633919/how-install-postgresql-9-4

Reply:
and see https://www.postgresql.org/download/linux/ubuntu/